package.json

```diff
@@ -29,13 +29,13 @@
   },
   "dependencies": {
     "cli-usage": "^0.1.1",
-    "clone": "^0.1.18",
+    "clone": "^1.0.2",
     "growly": "^1.2.0",
-    "lodash.clonedeep": "^3.0.0",
+    "lodash.clonedeep": "^3.0.1",
```

Comment:
Well, we are using lodash.clonedeep and set the compatible version to ^3.0.0, so 3.0.1 should be compatible as well, following the semver standard.